mcp/health-tool-mcp/server.py

```
@@ -0,0 +1,1357 @@
+    
+    # Common Windows installation paths
+    common_paths = [
+        r"C:\Program Files\Ballerina",
+        r"C:\Program Files (x86)\Ballerina",
+        os.path.expanduser(r"~\AppData\Local\Programs\Ballerina"),
+    ]
+    
+    for base_path in common_paths:
+        if os.path.exists(base_path):
+            # Look for bal.exe in bin subdirectories
+            for root, dirs, files in os.walk(base_path):
```

Ruff (0.15.12)

[warning] Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)